dev-lang/vala/vala-0.44.1.recipe

```diff
@@ -0,0 +1,123 @@
+SUMMARY="An object-oriented programming language that uses the GObject system"
+DESCRIPTION="Vala is a new programming language that aims to bring modern \
+programming language features to GNOME developers without imposing any \
+additional runtime requirements and without using a different ABI \
+compared to applications and libraries written in C."
+HOMEPAGE="https://wiki.gnome.org/Projects/Vala"
+COPYRIGHT="2006-2018 Jürg Billeter
+	2006-2018 Raffaele Sandrini"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://download.gnome.org/sources/vala/${portVersion%.*}/vala-$portVersion.tar.xz"
+CHECKSUM_SHA256="96a8c1415c6484d7d6b837ae263af7c50465c7423bfc8eea5046f3eadfa42fe2"
+PATCHES="vala-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="0.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+portVers="${portVersion%.*}"
+
+PROVIDES="
+	vala$secondaryArchSuffix = $portVersion
+	cmd:vala$commandSuffix
+	cmd:vala_$portVers$commandSuffix
+	cmd:vala_gen_introspect$commandSuffix
+	cmd:vala_gen_introspect_$portVers$commandSuffix
+	cmd:valac$commandSuffix
+	cmd:valac_$portVers$commandSuffix
+	cmd:valadoc$commandSuffix
+	cmd:valadoc_$portVers$commandSuffix
+	cmd:vapigen$commandSuffix
+	cmd:vapigen_$portVers$commandSuffix
+	lib:libvala_$portVers$secondaryArchSuffix = $libVersionCompat
+	lib:libvaladoc_$portVers$secondaryArchSuffix = $libVersionCompat
+	lib:libvalaccodegen$secondaryArchSuffix = $portVersion
+	devel:libvala_$portVers$secondaryArchSuffix = $libVersionCompat
+	devel:libvaladoc_$portVers$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcdt$secondaryArchSuffix
+	lib:libcgraph$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgmodule_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libgvc$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	# seems like vala needs a vala version to be installed
+	lib:libvalaccodegen$secondaryArchSuffix >= 0.42
```

Comment:
This makes no sense. When vala needs vala to build, then it's in BUILD_REQUIRES, and probably devel:libvalaccodegen$secondaryArchSuffix

Reply:
Will look further into it, this comes from their homepage in the requirements: Vala 0.16.1 or later (for the initial build)

Reply:
Removed ...